SETUP.md

```diff
@@ -70,9 +90,15 @@ You can do Code.org development using OSX, Ubuntu, or Windows (running Ubuntu in
         ```
     1. close and reopen your current terminal window
     1. make sure that `ulimit -n` returns 8192
-1. install the Xcode Command Line Tools:
+1. Install the Xcode Command Line Tools:
     1. `xcode-select --install`
-
+    
+    <details>
+              <summary>If it complains about command line tools already installed</summary>
+              
+              If it complains "xcode-select: error: command line tools are already installed, use "Software Update" to install updates" check to make sure XCode is downloaded and up to date manually.
```

Comment:
Suggest: put the error message in a code block.

Reply:
Updated